run.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+cd /Users/acflorea/phd/k7m/
```

Comment:
Avoid absolute paths related to your computer

Reply:
Please create a folder called `wrs` where the scripts and code related to wrs is stored